prov/gni/src/gnix_rma.c

```diff
@@ -79,17 +114,20 @@ static int __gnix_rma_send_completion(struct gnix_fid_ep *ep,
 				  "_gnix_cntr_inc() failed: %d\n", rc);
 	}
 
-	return rc;
+	return FI_SUCCESS;
```

Comment:
We could return an error here but there would be no way for it to propagate up to a user.  This function would be called inside the progress loop.